src/components/pages/get-a-quote/form.svelte

```diff
@@ -64,17 +65,17 @@
       character = 'attach';
     }
   } else if (lastChangedInput === 'budget') {
-    if (['15.000€ to 50.000€'].includes(budget)) {
+    if (budgetRange[0].includes(budget)) {
       character = 'budget10';
-    } else if (['50.000€ to 100.000€'].includes(budget)) {
+    } else if (budgetRange[1].includes(budget)) {
       character = 'budget25';
-    } else if (['100.000€ to 200.000€'].includes(budget)) {
+    } else if (budgetRange[2].includes(budget)) {
       character = 'budget25';
-    } else if (['200.000€ to 300.000€'].includes(budget)) {
+    } else if (budgetRange[3].includes(budget)) {
       character = 'budget50';
-    } else if (['300.000€ to 400.000€'].includes(budget)) {
+    } else if (budgetRange[4].includes(budget)) {
       character = 'budget50';
-    } else if (['400.000€ and above'].includes(budget)) {
+    } else if (budgetRange[5].includes(budget)) {
       character = 'budget100';
     }
```

Comment:
I think this is way too repetitive, can you refactor to a more dynamic solution?

For example:
```ts
const characters = ["budget10", "budget25", "budget50"]
let character = characters[0] // Default selection

bugdetRange.forEach((bRange, idx) => {
  if (bRange.includes(budget)) {
    character = characters[idx]
  }
})
```

Does it make sense?

Reply:
Makes sense thx, I will create a issue for this refactor, since right now I need to publish this in order not not break on release